pandas/core/common.py

```diff
@@ -479,6 +481,43 @@ def _dict_compat(d):
                 for key, value in iteritems(d))
 
 
+def _standardize_mapping(into):
+    """
+    Helper function to standardize the supplied mapping so it can
+    be passed to the ``Series.to_dict()`` and ``DataFrame.to_dict()``
+
+    Parameters
+    ----------
+    into : instance or subclass of collections.Mapping
+        The argument supplied to ``to_dict``.  Must be a class, an
+        initialized collections.defaultdict, or an empty instance
+        of a collections.Mapping subclass.
+
+    Returns
+    -------
+    mapping : a collections.Mapping subclass or other constructor
+        a callable object that can accept an iterator to create
+        the desired Mapping.
+
+    """
+    if not inspect.isclass(into):
```

Comment:
The reason I wrote it this way is because I imagined someone might pass a pre-populated mapping to this function and expect the series or frame to be appended to what was already in the passed mapping.  The error was to make it explicit that this would not happen.

Also happy to simplify this function a little  - I'll remove this logic if you don't think its worth checking an empty mapping.